pisa/stages/xsec/dis_sys.py

```diff
@@ -0,0 +1,213 @@
+"""
+Stage to apply pre-calculated DIS uncertainties
```

Comment:
please add a reference so others know where all of this comes from (a presentation, a paper, etc.); also references to author(s) are nice here so people know who to ask. And not just you, but presumably Maria and Juan Pablo. You can answer coding questions, but they would be the first people to turn to to answer physics questions. Readers of the code need to know those things (and the text you put on the PR will be difficult if impossible to find).

Reply:
done